### ISSUE TITLE
Implement quantile neural network estimator

### ISSUE THREAD

**Issue by saattrupdan** [saattrupdan/doubt#4]
For now work with `PyTorch` models, work with two quantiles and use the standard quantile loss function.
